engine/class_modules/sc_mage.cpp

```diff
@@ -164,14 +164,14 @@ struct buff_stack_benefit_t
   const buff_t* buff;
   std::vector<benefit_t*> buff_stack_benefit;
 
-  buff_stack_benefit_t( const buff_t* _buff, const std::string& prefix ) :
+  buff_stack_benefit_t( const buff_t* _buff, util::string_view prefix ) :
```

Comment:
I've already swapped the mage module to `std::string_view`

Reply:
I didn't consciously look at the mage file changes in summary and back off after seeing that everything was already adjusted.

Would you like me to revert the change here in buff_stack_benefit, or should I keep it (with the formatting changes suggested below)?

Reply:
Oh, definitely keep the change. I just wanted `std` namespace instead of `util` (since that's what the mage module now uses).

Reply:
Oh. I haven't thought about that, I hoped we would eventually change everything in a synced,automated fashion instead of having both worlds. But if mage module already uses that I can of course adjust.